src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -157,11 +155,11 @@ private static void addDailyContributionForNewDate(
     /**
      * Gets the starting point of the {@code current} date.
      *
-     * @return the {@code current} date if it is equal to the {@code ARBITRARY_FIRST_COMMIT_DATE_UTC} adjusted to the
+     * @return the {@code current} date if it is equal to the {@code ARBITRARY_FIRST_COMMIT_DATE} adjusted to the
      * timezone given by {@code zoneId}. Otherwise, return a {@link LocalDateTime} adjusted to have a time of 00:00:00.
      */
     private static LocalDateTime getStartOfDate(LocalDateTime current, ZoneId zoneId) {
-        if (current.equals(ARBITRARY_FIRST_COMMIT_DATE_UTC.withZoneSameInstant(zoneId).toLocalDateTime())) {
+        if (current.equals(SinceDateArgumentType.getArbitraryFirstCommitDateConverted(zoneId))) {
```

Comment:
This check seems to be performed a lot, would it make sense to create TimeUtil::isArbitratyFirstCommitDate[UtcLocal] or something to that effect?